src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -344,12 +344,12 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         enhancer.setSuperclass(delegate.getClass())
         enhancer.setCallback([
 
-                invoke: {Object proxy, Method method, Object[] args ->
-                    if ("onNext" == method.name) {
-                        logger.info(args[0].stream)
-                    }
-                    method.invoke(delegate, args)
+            invoke: {Object proxy, Method method, Object[] args ->
+                if ("onNext" == method.name) {
+                    logger.quiet(args[0].toString())
```

Comment:
@rfhayashi looks like you're right. Let me go back and have a look at things.

`toString(): BuildResponseItem[stream=Step 1 : FROM ubuntu:12.04
,status=<null>,progressDetail=<null>,progress=<null>,id=<null>,from=<null>,time=0,errorDetail=<null>,error=<null>]

stream: Step 1 : FROM ubuntu:12.04`